tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/TemplatesTest.cs

```diff
@@ -1250,6 +1250,22 @@ public void TestCustomFunction2()
             Assert.AreEqual(6.0, evaled);
         }
 
+        [TestMethod]
+        public void TestInjectLG()
+        {
+            var templates = Templates.ParseFile(GetExampleFilePath("./InjectionTest/inject.lg"));
+            
+            var evaled = templates.EvaluateText("${foo.bar()}");
```

Comment:
maybe use Expression to test it is more suitable.
```
Expression.Parse("foo.bar()").TryEvaluate(xxx)
```

Reply:
That's the right way to test it. Thanks